build/assets/style.csl

```diff
@@ -16,6 +16,10 @@
   <macro name="author">
     <names variable="author">
       <name initialize="false" initialize-with="" et-al-min="12" et-al-use-first="10" et-al-use-last="true"/>
+      <substitute>
+        <names variable="editor"/>
+        <text macro="venue"/>
+      </substitute>
```

Comment:
From http://docs.citationstyles.org/en/stable/specification.html#substitute

> Substituted variables are suppressed in the rest of the output to prevent duplication. An example, where an empty “author” name variable is substituted by the “editor” name variable, or, when no editors exist, by the “title” macro: